### PR TITLE
Unit tests fixes

### LINKED ISSUE #1887: ISOTP tests fails
This issue is a reminder of the discussion started in #1866 : ISOTP tests must check if they can be launched.

### PR DESCRIPTION
**Pull request by guedou**
This PR does several things:
- allows tests to be run without any network connectivity
- removes as useless test
- flags more tests using `tcpdump`
- documents using `tox` on a fresh Linux installation (that fixes #1887)

### CODE REVIEW

File: test/regression.uts [hunk 1]
```diff
@@ -36,8 +36,6 @@ assert not _version_checker(FakeModule2, (5, 143, 4))
 
 assert _version_checker(FakeModule3, (2, 4, 2))
 
-assert _version_checker(scapy, (2, 4, 2))
```
Comment (gpotter2) [inline test/regression.uts]: Is this failing ?

**Comment by guedou** [comment]
Not yet, but it will fail with v2.4.3.

File: test/regression.uts (hunk 1 above)
Comment (guedou) [reply test/regression.uts]: Commit removed =)

**Comment by guedou** [comment]
Auto-merging as it was approved by @gpotter2.

**Pull request merged**